### ISSUE TITLE
Improve CMake installation

### ISSUE THREAD

**Issue by Holzhaus** [ChristophHaag/SteamVR-OpenHMD#19]
I created an AUR packagei for easy installation on Arch Linux: https://aur.archlinux.org/packages/steamvr-openhmd-git/

In the process, I noticed that the installation process could use some improvement. `make install` does not work properly, which is why I needed to [use `install` to copy files by hand](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=steamvr-openhmd-git#n40).

Ideally, `make install` should be used to copy `bin`, `resources` and the `driver.vrdrivermanifest` file to `/usr/lib/steamvr/openhmd` or something like that. Then, users you just create a symlink to their SteamVR installation and everything works, e.g.:

    $ ln -s /usr/lib/steamvr/openhmd ~/.steam/steam/steamapps/common/SteamVR/drivers/openhmd

**Comment by ChristophHaag** [comment]
Yes, I did not care about installing at all so far because SteamVR can load registered plugins from anywhere.

I can look into how [SteamVR-OSVR](https://github.com/OSVR/SteamVR-OSVR) does it, because installing the package was easy in my AUR package that I of course misnamed as [osvr-steamvr](https://aur.archlinux.org/packages/osvr-steamvr-git/).
I believe SteamVR-OSVR registers the plugin during make install so it works without further user action.
